docs/kbs_attestation_protocol.md

```diff
@@ -474,6 +474,29 @@ When the KBC uses this token to request resources or services from a relying
 party service API, then the symmetric key used to encrypt the output payload can
 be wrapped with the provided `tee-pubkey`.
 
+### Set Attestation Policy
+User of KBS can set attestation verification policy with posting following endpoint:
+
+```
+/kbs/v0/attestation-policy
+```
+
+The payload of the POST request should like:
+
+```json
+{
+    "type": "rego",
+    "policy_id": "default",
+    "policy": <base64encoded policy>
+}
+```
+
+Where `type` specified the policy format, `policy_id` specified the ID of the policy
+and `policy` is the base64 encoded policy content.
+
+POST to this endpoint needs user identity authentication with carrying JWT (JSON Web Token)
+signed by the user's private key in the Request.
```

Comment:
So the attestation policy is global to the whole KBS? Are we expecting the policy to describe e.g. different policies for different resources?

Reply:
The policy here is the policy for verifying evidence, and in the RATS model, it should be the policy of the Verifier. And the policy you mentioned should be a policy of the rely party, which distributes different confidential resources based on the attestation results.

I think it's best for us to separate these two policies, although we haven't implemented the latter yet.

Reply:
Ok, that makes sense to me. Thanks for the clarification.